release-notes.md

```diff
@@ -1,5 +1,15 @@
 # Release notes
 
+## October 7, 2019
+
+The following updates were made in this release:
+
+**iOS Mobile Services 1.0.3**
+
+* Fullscreen messages now use WKWebViews from WebKit.framework.
```

Comment:
Please reword as follows:
"Full-screen messages now use WKWebViews from the WebKit.framework."